core/src/main/java/org/apache/accumulo/core/clientImpl/NamespaceOperationsImpl.java

```diff
@@ -172,9 +172,10 @@ public void rename(String oldNamespaceName, String newNamespaceName)
   }
 
   private boolean validProperty(final String property, final String value) {
-    if (!property.startsWith(Property.TABLE_HDFS_POLICY_PREFIX.getKey()))
-      return true;
-    return Property.isValidHdfsPolicy(property, value);
+    if (property.equals(Property.TABLE_STORAGE_POLICY.getKey())
```

Comment:
Not sure how well this would work.  Another possible way to accomplish this may be with a new Accumulo property type.  Each config property has type and those types have validation methods.

Reply:
do you think validating property values here in the client code is ok, or should that be left for the master to do?

Reply:
Ideally we do it in both places.  Validation on the client side is nice for fail fast and clear error messages.  Validation on the server side is nice for the case where its not the Accumulo client software making the RPCs

Reply:
Ok, I think I'm doing the validation properly now (since I just copied the validation code from the server side).  Had some fun with spotbugs/test code interaction.